client/src/components/App/Content/InvestigationForm/TabManagement/PersonalInfoTab/PersonalInfoTab.tsx

```diff
@@ -348,20 +264,66 @@ const PersonalInfoTab: React.FC = (): JSX.Element => {
                         </RadioGroup>
                     </FormControl>
                 </Grid>
+                {
+                    personalInfoStateContext.personalInfoData.relevantOccupation === 'מערכת החינוך' &&
+                        <Grid item xs={2}>
+                            <Autocomplete
+                                options={Array.from(cities, ([name, value]) => ({ name, value }))}
+                                getOptionLabel={(option) => option.value.displayName}
+                                inputValue={personalInfoStateContext.personalInfoData.educationOccupationCity}
+                                onInputChange={(event, newInputValue) => {
+                                    handleChangeField(PersonalInfoDataContextFields.EDUCATION_OCCUPATION_CITY, newInputValue)}
+                                }
+                                onChange={(event, newValue) => {
+                                    newValue && getEducationSubOccupations(newValue.value.displayName);
+                                }}
+                                renderInput={(params) =>                     
+                                <CircleTextField
+                                    {...params}
+                                    id={PersonalInfoDataContextFields.CITY}
+                                    placeholder={'עיר המצאות המוסד'}
+                                    value={personalInfoStateContext.personalInfoData.educationOccupationCity}
+                                />}
+                            />
+                        </Grid>
+                }   
                 <Grid item xs={3}>
-                    <CircleSelect
-                        value={personalInfoStateContext.personalInfoData.institutionName}
-                        label={INSERT_INSTITUTION_NAME}
-                        options={INSTITUTION_OPTIONS}
-                        className={classes.institutionName}
-                        onChange={(event) => {
-                            handleChangeField(PersonalInfoDataContextFields.INSTITUTION_NAME, event.target.value);
-                        }}
-                    />
+                    <Collapse in={personalInfoStateContext.personalInfoData.relevantOccupation !== 'לא עובד'}>
+                    {
+                        (personalInfoStateContext.personalInfoData.relevantOccupation === 'כוחות הביטחון' ||
```

Comment:
Cant you just check if subOccupations is null or empty array? whatever you init it with